src/utils/discordEphemeralResponse.ts.ts

```diff
@@ -0,0 +1,18 @@
+import { InteractionResponseType } from "discord-interactions";
+import JSONResponse from "./JsonResponse";
+
+export const discordephemeralResponse = async (
+  reply: string,
+  discordToken: string
+): Promise<JSONResponse> => {
+  return new JSONResponse({
+    type: InteractionResponseType.CHANNEL_MESSAGE_WITH_SOURCE,
+    data: {
+      content: reply,
+      flags: 64,
+    },
+    Headers: {
```

Comment:
are you sure its `Headers` and not `headers`

Reply:
That was my blunder, `ephemeral` messages do not require any headers. 
Thanks for pointing out.